src/General/Badge/BadgeStyle.ts

```diff
@@ -5,13 +5,15 @@ import { BadgeVariant } from './BadgeVariant';
 export const BadgeContainer = styled.div<BadgeContainerProps>`
   position: relative;
   display: inline-flex;
+  align-items: center;
```

Comment:
@Kevintirta , why the changes in the styles?

Reply:
seems that the content of the label does not align vertically but, the new changes also break the superscript type. I have screenshotted the component below